### ISSUE TITLE
doubt with 1013, 1028 and 1003 warnings

### ISSUE THREAD

**Issue by Helmschmid** [ShikyoKira/Project-New-Reign---Nemesis-Main#483]
Hello, today I moved to nemesis from FNIS for Skyrim SE for Steam. I instaled the engine with no problems, updated the engine but after lunching I got 6004 warning notifications. Is it normal to have that many? did I do something wrong? can it be fixed? I tested the game and didn't noticed any animation errors so far so I'm very confused right now

**Comment by Heisumu** [comment]
I'm experiencing the same but with 6795 warnings. 
Actually, this is the same issue already brought up in the issue #222 , it was said fixed and closed, but I read the whole passage and still confused about how to fix it.
I'm skeptical about the engine file, as every time I try to use the 0.84 version it crashes, the only solution I used is to use a 0.83version and update instead.

**Comment by RichDavis1** [comment]
I am experiencing the same issue. Every single mod reports an error and I do not have any files located in ``Nemesis_Engine\temp_behaviors\_1stperson`` or ``Nemesis_Engine\temp_behaviors\xml`` 

Some guidance would be much appreciated. I'm installing by installing this as a mod in mo2 and opening in mo2 executables.

**Comment by CupiBo** [comment]
Yeah, same here.... it used to be fine, but after i clean started my mod setup it started to show this error. I tested it with mods and without and tried it with alot of deferent version of nemesis (except 0.84 because of the 6004 error), but nothing seems to work.
